python/paddle/v2/framework/tests/gradient_checker.py

```diff
@@ -76,6 +91,127 @@ def product(dim):
     return gradient_flat.reshape(tensor_to_check.get_dims())
 
 
+class GradientChecker(unittest.TestCase):
+    def check_grad(self,
+                   forward_op,
+                   input_vars,
+                   inputs_to_check,
+                   output_name,
+                   no_grad_set=set(),
```

Comment:
Do not use complex value as default argument.

Python's default argument is very very strange. Python default argument value is mutable.

See [here](http://effbot.org/zone/default-values.htm)